### PR TITLE
Johan/bevy 0 10

### PR DESCRIPTION
**Pull request by joverwey**
Hi, I had to make some change locally to be compatible with Bevy 0.10.

I was also able to remove some code. The latest Bevy plugin code will only update the AssetServer if it hasn't already been initialized. So it is ok to continue calling the DefaultPlugin.

Something I did notice is that the call to surf::get() is very slow. About 20 seconds. This is when running natively, not wasm.
Have you noticed the same or did I break something?

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,9 +26,4 @@ wasm-bindgen-futures = "0.4"
 web-sys = {version = "0.3.22", default-features = false}
 
 [dev-dependencies]
-bevy = {version = "0.9", default-features = false, features = [
-  "bevy_asset",
-  "png",
-  "render",
-  "x11", # GitHub Actions runners don't have libxkbcommon installed, so can't use Wayland
-]}
+bevy = "0.10"
```
Comment (johanhelsing) [inline Cargo.toml]: Please don't include more features than necessary

**Review by johanhelsing — COMMENTED** [review]
Nice to see so many red lines. Not sure what's up with surf taking so long. Never happened to me. I can give it a check tomorrow and see if it happens for me as well.

File: Cargo.toml (hunk 1 above)
Comment (johanhelsing) [reply Cargo.toml]: Oh, i didn't realize it was dev deps at first... Still it will build faster in ci and dev without all the extra stuff

**Comment by johanhelsing** [comment]
No slow requests for me... `web_image` example loads in less than a second.

Are you also seeing this on the 0.9 version?

Also: to fix the lints, you just need to run `cargo fmt`

**Comment by joverwey** [comment]
> Owner

It is slow on the 0.9 version too. Must be something with my network setup.

**Comment by johanhelsing** [comment]
Sorry, I kind of forgot about this PR... Merging since there are no regressions. If slow requests turns out to be a problem specific to this crate, let's open a new issue.

**Pull request merged**